.github/workflows/nightly-pipeline.yaml

```diff
@@ -149,6 +150,41 @@ jobs:
           client_payload: ${{ toJSON(fromJSON(needs.get-run-info.outputs.obj).payloads.kvikio) }}
           propagate_failure: true
           trigger_workflow: true
+  cuvs-build:
+    needs: [get-run-info]
```

Comment:
Looks like `cuvs` depends on `raft` (https://github.com/rapidsai/cuvs/blob/branch-24.04/conda/recipes/libcuvs/meta.yaml#L60), so this needs to be reordered

Reply:
Ah, will address that